appveyor.yml

```diff
@@ -0,0 +1,85 @@
+#matrix:
+#  fast_finish: true
+
+environment:
+  global:
+    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+    # /E:ON and /V:ON options are not enabled in the batch script intepreter
+    # See: http://stackoverflow.com/a/13751649/163740
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\appveyor\\run_with_env.cmd"
+
+    # 1. Generated a token for appveyor at https://anaconda.org/quantopian/settings/access with scope api:write.
+    #   Can also be done via anaconda CLI with
+    #     $ anaconda auth --create --name my_appveyor_token
+    # 2. Generated secure env var below via appveyor's Encrypt data tool at https://ci.appveyor.com/tools/encrypt.
+    #   See https://www.appveyor.com/docs/build-configuration/#secure-variables.
+    ANACONDA_TOKEN:
+      secure: "ryxTeOgjDvz6Y25wr29GcUm1Y7iCOeNX96t1iXIp4bQzWCQszJ68+M6Zy6Kpg0C6"
```

Comment:
I'm not certain - Should we generate new tokens for this repo?

Reply:
removed this value - realized we don't need it if zipline is handling conda builds, but if we end up having this repo handle its own conda builds, might be worth revisiting